src/views/guidelines/guidelines.jsx

```diff
@@ -0,0 +1,45 @@
+var React = require('react');
+var render = require('../../lib/render.jsx');
+var Page = require('../../components/page/page.jsx');
+
+require('./guidelines.scss');
+
+var Guidelines = React.createClass({
+    type: 'Guidelines',
+    render: function () {
+        return (
+            <div className="body">
+                <div className="top">
```

Comment:
We have a component called [`Box`](https://github.com/LLK/scratch-www/blob/develop/src/components/box/box.jsx) which was created to have reusable styling for pages like this, where everything is contained within a bordered box (*see: "What's Happening" on the homepage, or the [`cards`](https://github.com/LLK/scratch-www/blob/develop/src/views/cards/cards.jsx) page*). I think we should re-use that component here.